fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -1264,6 +1266,25 @@ public void rollEditLog() {
         journal.rollJournal();
     }
 
+    private synchronized <T extends Writable> void logEdit(short op, List<T> entries) throws IOException {
+        List<JournalBatch> batches = new ArrayList<>();
+        JournalBatch batch = new JournalBatch(35);
+        for (T entry : entries) {
+            // the number of batch entities to less than 32 and the batch data size to less than 640KB
+            if (batch.getJournalEntities().size() >= 32 || batch.getSize() >= 640 * 1024) {
+                batches.add(batch);
```

Comment:
How about writing to journal directly, like:

```
journal.write(batch);
batch = new JournalBatch(35);
```

Reply:
fixed